es-ds-docs/pages/molecules/verification-code.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+if ($prism) {
+    /* eslint-disable import/no-webpack-loader-syntax, import/no-self-import */
+    const compSource = await import('@energysage/es-ds-components/components/es-verification-code.vue?raw');
+    const docSource = await import('./verification-code.vue?raw');
+    /* eslint-enable import/no-webpack-loader-syntax, import/no-self-import */
+
+    compCode.value = $prism.normalizeCode(compSource.default);
+    docCode.value = $prism.normalizeCode(docSource.default);
+    $prism.highlight();
+}
+
+const propTableRows = [
+    ['v-model', 'String', 'n/a', 'Required. The v-model directive binds the otpinput to a data property.'],
```

Comment:
`charCount` is also a prop we should document here

Also, we can probably just say "input" instead of "otpinput", as "otpinput" is an internal implementation detail of this component that consumers of this component won't have to worry about.